src/RelayServer.ts

```diff
@@ -214,13 +219,14 @@ export class RelayServer extends EventEmitter {
             );
         }
 
-        // Check the relayWorker (todo: once migrated to multiple relays, check if exists)
+        // Check the feesReceiver (todo: once migrated to multiple relays, check if exists)
         if (
-            req.relayRequest.relayData.relayWorker.toLowerCase() !==
-            this.workerAddress.toLowerCase()
+            ![this.feesReceiver, this.workerAddress].includes(
+                req.relayRequest.relayData.feesReceiver.toLowerCase()
```

Comment:
We are comparing `feesReceiver` and `workerAddress` to `req.relayRequest.relayData.feesReceiver.toLowerCase()`. We probably need to apply the `toLowerCase` function to everything.
```suggestion
            ![this.feesReceiver.toLowerCase(), this.workerAddress.toLowerCase()].includes(
                req.relayRequest.relayData.feesReceiver.toLowerCase()
```

Reply:
this has been changed to something entirely different. Now we check againts the feesReceiver field in RelayServer.ts